src/Marten/Events/AppendEventFunction.cs

```diff
@@ -8,10 +8,12 @@ namespace Marten.Events
     public class AppendEventFunction : Function
     {
         private readonly EventGraph _events;
+        private readonly bool _useAppendEventForUpdateLock;
 
-        public AppendEventFunction(EventGraph events) : base(new DbObjectName(events.DatabaseSchemaName, "mt_append_event"))
+        public AppendEventFunction(EventGraph events, bool useAppendEventForUpdateLock = false) : base(new DbObjectName(events.DatabaseSchemaName, "mt_append_event"))
```

Comment:
Shouldn't the value of useAppendEventForUpdateLock be coming off of EventGraph? Why bother with the extra arg?